src/BizHawk.Bizware.BizwareGL/EDispMethod.cs

```diff
@@ -4,6 +4,7 @@ public enum EDispMethod
 	{
 		OpenGL = 0,
 		GdiPlus = 1,
-		SlimDX9 = 2,
+		D3D9 = 2,
+		D3D11 = 3,
```

Comment:
Am I seeing it right that this isn't actually hooked up anywhere (yet)?

Reply:
Ah right, that was a D3D11 backend I was working on but didn't complete yet, wasn't intended to be part of this PR.